Applications/CLI/ogs.cpp

```diff
@@ -85,14 +78,9 @@ int main(int argc, char *argv[])
 {
 	using ConfigTree = boost::property_tree::ptree;
 
-#ifdef USE_MPI
-	MPI_Init(&argc, &argv);
-#endif
-
-#ifdef USE_PETSC
-	char help[] = "ogs6 with PETSc \n";
-	PetscInitialize(&argc, &argv, nullptr, help);
-#endif
+	// Initialize MPI, PETSc, LIS or any other database from third party
+	// packages.
```

Comment:
I would remove this comment and also the one on `OgsFinalize()` because the name already implies what the methods do and the same comment is already given in the method documentation.